packages/lit-html/package.json

```diff
@@ -14,114 +14,142 @@
   "type": "module",
   "exports": {
     ".": {
+      "types": "./development/lit-html.d.ts",
```

Comment:
So does this take the place of having to copy types to the root of the package? Can we remove treemirror now? Or do we still need it for back-compat with older TypeScript versions.